src/AudioUnit/AudioUnit.cs

```diff
@@ -403,7 +409,7 @@ public static uint GetCurrentInputDevice ()
 			var theAddress = new AudioObjectPropertyAddress (
 				AudioObjectPropertySelector.DefaultInputDevice,
 				AudioObjectPropertyScope.Global,
-				AudioObjectPropertyElement.Master);
+				AudioObjectPropertyElement.Main);
```

Comment:
@spouliot  do we trust apple to do the correct things, or should we just use master ?

Reply:
both enum members have the same `0` value so there's no risk for this one